examples/calculator.cc

```diff
@@ -22,6 +22,10 @@ void Calculator::Print(const std::string& message) {
   printf("%s\n", message.c_str());
 }
 
+void Calculator::CamelCase() {
```

Comment:
Could you please rename Equals() method to IsEquals() without adding this new method?
(We wanted to use IsEuqals() but used Equals() due to this camel case problem.)

Reply:
Okay I will do this